server/src/routes/api/v1/userActivityRouter.js

```diff
@@ -0,0 +1,28 @@
+import express from "express"
+
+import { User } from "../../../models/index.js"
+
+const userActivityRouter = new express.Router({ mergeParams: true })
+
+userActivityRouter.get("/", async (req, res) => {
```

Comment:
instead of a new router, add this endpoint to the existing usersRouter for consistency